src/main/features/claim/views/check-and-send.njk

```diff
@@ -60,8 +60,8 @@
       {{ row('Interest rate:', t('I don’t want to claim interest'), paths.interestPage.uri )}}
     {% endif %}
 
-    {{ row('Claim fee:', claimAmountTotal.feeAmount | numeral) }}
-    {{ row('Total amount you’re claiming:',  claimAmountTotal.totalAmount | numeral) }}
+    {{ row("Claim fee:", claimAmountTotal.feeAmount | numeral) }}
```

Comment:
No more double quotes please.

Reply:
Yes of course since the new introduction of template rules